augustocfreitas/ChronoConquest
# feat: Initial Project Setup and Repository Structure

## Description

This pull request establishes the foundation of the project. It includes the setup of the GitHub repository, initialization of the project using a package manager, and the creation of a standardized directory structure. Additionally, a dedicated `develop` branch has been created to manage the development workflow.

## Changes

- Initialized the GitHub repository with essential GitHub community standards such as CONTRIBUTING.md, ISSUE_TEMPLATE.md, etc.
- Set up the package manager and installed initial dependencies.
- Created a well-defined directory structure for the project’s core functionality.
- Established the `develop` branch following the Gitflow workflow for future feature integration and releases.

## Motivation & Context

Laying the groundwork is crucial for a smooth development experience and ensures that the project is scalable and maintainable from the onset. This setup will also facilitate easier onboarding for new contributors.

## How to Test

Since this PR is for initial setup, there's no functional code to test yet. However, you can review the directory structure and ensure that the repository follows best practices.


## Checklist

- [x] The GitHub repository has been properly set up with all initial files.
- [x] The project has been initialized with the chosen package manager.
- [x] The directory structure follows our agreed-upon standard.
- [x] The `develop` branch has been created following the Gitflow principles.
